pkgs/applications/radio/osmo-rtl-sdr/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchgit
+, cmake
+, pkg-config
+, libusb1
+}:
+
+stdenv.mkDerivation rec {
+  pname = "osmo-rtl-sdr";
+  version = "2.0.1";
+
+  src = fetchgit {
```

Comment:
Any reason your not using `fetchFromGitea`? (:

Reply:
Nope, made the switch!